src/docs/installing/installing-linux.md

```diff
@@ -14,19 +14,15 @@ Only Linux kernel 6.4 or newer is supported.
 ## Recommended Distributions
 
 ::: warning
-Ubuntu and Ubuntu-based distributions **may have issues** and are **not supported**.
-:::
-
-::: warning
-Debian versions older than Debian 12 (Bookworm) are **not supported**. Debian 12 (Bookworm) requires a custom kernel, the [audio script](#fixing-audio) will automatically install it for you.
+Old (>1 year) LTS-Distributions distributions **likely have issues** and may not be supported.  
```

Comment:
Right here is where it says may not be supported

Reply:
just saw what u meant aswell. fixed.